src/utils/PolygonConnector.cpp

```diff
@@ -9,260 +9,97 @@
 namespace cura 
 {
 
-Polygons PolygonConnector::connect()
+PolygonConnector::PolygonConnector(const coord_t line_width)
+: line_width(line_width - 5) // a bit less so that consecutive lines which have become connected can still connect to other lines
```

Comment:
This particular magic number has been removed now, actually :)

It was a relic of the old implementation. It had a minimum connection length as well, refusing to connect things closer together than 1 line width. I'm not exactly sure why that minimum was in place, as that case should never really occur, unless the input polygons were already self-overlapping. In any case, it's not applicable to the case of variable line width anyway.

It was also causing the test cases to fail because this changes the line width.